test/integration/visitor_can_view_items_test.rb

```diff
@@ -25,10 +25,12 @@ class VisitorCanViewItemsTest < ActionDispatch::IntegrationTest
     item_3 = Chip.create(name: "Dang Coconut", price: 17, description: "Dang, these are good", oil_type_id: category_2.id)
     item_4 = Chip.create(name: "Lard Yummies", price: 19, description: "Chock Full of Lard", oil_type_id: category_1.id)
 
-    visit '/oil_types'
+    visit oil_types_path
 
     click_link "Lard"
 
+    assert current_path, '/oil_types/lard'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.